setup.py

```diff
@@ -35,18 +34,17 @@
         'Programming Language :: Python :: 3.7',
         'Topic :: Software Development :: Build Tools'
         'Topic :: Software Development :: Embedded Systems',
-    ),
-    description="Mbed Tools: Target information for building Mbed OS",
-    keywords="Arm Mbed OS MbedOS build target platform board module",
+    ],
+    description='Mbed Tools: Target information for building Mbed OS',
+    keywords='Arm Mbed OS MbedOS build target platform board module',
     include_package_data=True,
     install_requires=requirements,
     license='Apache 2.0',
     long_description_content_type='text/markdown',
     long_description=long_description,
     name=PACKAGE_NAME,
-    package_dir={'': SOURCE_DIR},
-    packages=find_packages(SOURCE_DIR),
+    packages=[SOURCE_DIR],
     python_requires='>=3.6,<4',
-    url="https://github.com/ARMmbed/%s" % PACKAGE_NAME,
+    url=f'https://github.com/ARMmbed/{PACKAGE_NAME}',
     version=__version__,
 )
```

Comment:
Could we also add entry points so that we don't need to always do `python -m ...`:
e.g.
    entry_points=dict(
        console_scripts=[
            "mbed_targets= mbed_targets.mbed_target:main",
            "mbedtargets = mbed_targets.mbed_target:main",
            "mbed-targets = mbed_targets.mbed_target:main",
            "mt = mbed_targets.mbed_target:main",
        ]

See https://setuptools.readthedocs.io/en/latest/setuptools.html
https://setuptools.readthedocs.io/en/latest/setuptools.html#automatic-script-creation
    ),

Reply:
If we were to do it, I'd rather put this in a separate PR.

- Do we really want this package to create console entry points though? My understanding was that there would be `mbed` (from `mbed-cli`) as an entry point? Like so https://click.palletsprojects.com/en/7.x/setuptools/ .
- What is `mbed_targets:mbed_target:main`? I can't find it.

Reply:
was an example of an entry point

Reply:
I had the impression we would also be able to use the tools separately but I may have misunderstood after all. 
Since this PR is about fixing `setup.py`, I though we should tackle all use cases with regards to it but would understand if we consider it as good enough for now